Automattic/wp-calypso
# Framework: Use i18n-calypso Jed v1.1 branch

**Don't merge** This is to demonstrate that https://github.com/Automattic/i18n-calypso/pull/72 works. Once that is merged and released, we need to bump the version here in `package.json`.

### Testing Instructions

* Verify that i18n still works
* Try switching languages
* In the browser's network tab, verify that instead of e.g. `de.js` and `de.json`, we're now loading `de-v1.1.js[on]`.
